Makefile

```diff
@@ -54,6 +54,7 @@ PASSWORD_STORE_DIR?=${HOME}/.paas-pass
 globals:
 	$(eval export AWS_DEFAULT_REGION=eu-west-1)
 	$(eval export PASSWORD_STORE_DIR=${PASSWORD_STORE_DIR})
```

Comment:
is this needed? is this expected to be different pass than the one used for datadog API keys?